core/src/main/java/com/novoda/accessibility/Service.java

```diff
@@ -0,0 +1,23 @@
+package com.novoda.accessibility;
+
+/**
+ * An enumeration of all the known accessibility services and their fully qualified names.
+ * <p>
+ * These are the values returned from android.accessibilityservice.AccessibilityServiceInfo#getId().
+ */
+public enum Service {
+
+    TALKBACK("com.google.android.marvin.talkback/.TalkBackService"),
+    SWITCH_ACCESS("com.google.android.marvin.talkback/com.android.switchaccess.SwitchAccessService"),
+    SELECT_TO_SPEAK("com.google.android.marvin.talkback/com.google.android.accessibility.selecttospeak.SelectToSpeakService");
+
+    private final String qualifiedName;
```

Comment:
There is an Android class called `ComponentName` to represent this. Here String is just enough but I wonder if it would make sense to use `ComponentName` instead.

Reply:
:+1: I'll rename to component name - maybe that's more accurate than qualified name since maybe qualified name is a specific term to refer to package + classname, whereas this is the app's package name + qualified name.

Not sure about swapping string for the typed one though - though it's clearer what the value is, it'll nuke the tests: the enum will not load because it'll be using an un-stubbed android class.

Reply:
commit 65ab957fe978d4b64f2f89972adf5b91f69aca7d
Author: Ataul Munim <hello@ataulm.com>
Date:   Wed Jan 31 07:30:02 2018 +0000

    Rename Service.qualifiedName to flattenedComponentName